nmostesting/mocks/Node.py

```diff
@@ -211,6 +212,16 @@ def parse_sdp(self, sdp_data):
                 sdp_param_leg["source_ip"] = filter_line.group(2)
             else:
                 sdp_param_leg["source_ip"] = None
+
+            format_line = re.search(r"a=fmtp:(\S*\s*)(.*)", sdp_data)
+
+            if format_line and format_line.group(2):
+                #  Handle parameter keys that have no value, e.g. 'interlace' and set the value to 'True'
+                sdp_param_leg["format"] = {key: ''.join(value) if len(value) > 0 else 'True'
+                                           for key, *value in
+                                           [item.split('=') for item in re.split(';\\s', format_line.group(2))]
```

Comment:
I think in the spirit of not getting hung up on the white space at this point in the test code, given the inconsistencies in this regard between the various RFC and ST 2110 revisions, how about:

```suggestion
                                           [item.split('=') for item in re.split(r'[ \t]*;[ \t]*', format_line.group(2))]
```

I use `[ \t]` rather than `\s` to be clear that newline chars are not included.